### PR TITLE
Add triggers lava mistake

### PR DESCRIPTION
**Pull request by latedude2**

### CODE REVIEW

File: Assets/Scripts/PlayerMovementPlayback.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ void FixedUpdate()
             if(line != null)
             {
                 string[] data = line.Split(' ');
-                if (!data[0].Equals("EndTaskCompleted") && data.Length > 1) //If this line has coordinates (not the end task tag)
+                if (!data[0].Equals("EndTaskCompleted") && data.Length > 2) //If this line has coordinates (not the end task tag)
```
Comment (Jokubas126) [inline Assets/Scripts/PlayerMovementPlayback.cs:40]: more thorough explanation of what that number means is needed I think so we all understand

File: Assets/Scripts/ReactToTeleportTriggers.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        {
+            teleportTriggers.Add(child.GetComponent<TeleportTrigger>());
+        }
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
```
Comment (Jokubas126) [inline Assets/Scripts/ReactToTeleportTriggers.cs]: not needed

File: Assets/Scripts/ReactToTeleportTriggers.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    private void OnTriggerEnter(Collider other)
+    {
+        TeleportTrigger teleportTrigger = other.GetComponent<TeleportTrigger>();
+        if (teleportTrigger != null)
+        {
+            //gameObject.GetComponent<FirstPersonController>().enabled = false;
+            gameObject.transform.position = teleportTrigger.teleportDestination.transform.position;
+            //gameObject.GetComponent<FirstPersonController>().enabled = true;
```
Comment (Jokubas126) [inline Assets/Scripts/ReactToTeleportTriggers.cs]: do you need those commented lines?

File: Assets/Scripts/ShortcutTrigger.cs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ShortcutTrigger : MonoBehaviour
+{
+    public bool visited = false;
```
Comment (Jokubas126) [inline Assets/Scripts/ShortcutTrigger.cs:7]: do you need it have it Serialized? It doesn't change anything apart from showing it in script component in for an object ofc, but just for a good practice 🤔

File: Assets/Scripts/TeleportTrigger.cs [hunk 5]
```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TeleportTrigger : MonoBehaviour
+{
+    public GameObject teleportDestination;
```
Comment (Jokubas126) [inline Assets/Scripts/TeleportTrigger.cs]: what is that used for? maybe adding an explanatory comment would be good idea?

File: Assets/Scripts/ShortcutTrigger.cs (hunk 4 above)
Comment (latedude2) [reply Assets/Scripts/ShortcutTrigger.cs:7]: it nice to see it change in the editor for debuging and testing

**Review by Jokubas126 — APPROVED** [review]
very nic

**Pull request merged**